public/onetrust/consent/52635430-c9e6-4d7f-a265-73f7bd2d5dc0-test/52635430-c9e6-4d7f-a265-73f7bd2d5dc0-test.json

```diff
@@ -1 +1 @@
n","bo","sa","sb","bq","sc","br","bs","sd","bt","sg","bv","bw","sh","by","sj","bz","sl","sn","so","ca","sr","cc","ss","cd","st","cf","sv","cg","sx","ch","sy","ci","sz","ck","cl","cm","cn","co","cr","tc","td","tf","cu","tg","cv","th","cw","cx","tj","tk","tl","tm","tn","to","tr","tt","tv","tw","dj","tz","dm","do","ua","ug","dz","um","ec","us","eg","eh","uy","uz","va","vc","er","et","ve","vg","vi","vn","vu","fj","fk","fm","fo","wf","ga","gb","ws","gd","ge","gg","gh","gi","gl","gm","gn","gq","gs","gt","gu","gw","gy","xk","hk","hm","hn","ye","ht","id","il","im","in","io","za","iq","ir","zm","je","zw","jm","jo","jp","ke","kg","kh","ki","km","kn","kp","kr","kw","ky","kz","la","lb","lc","lk","lr","ls","ly","ma","md","me","mg","mh","mk","ml","mm","mn","mo","mp","mr","ms","mu","mv","mw","mx","my","mz","na","nc","ne","nf","ng","ni","np","nr","nu","nz","om","pa","pe","pf","pg","ph","pk","pm","pn"],"States":{},"LanguageSwitcherPlaceholder":{"default":"it"},"BannerPushesDown":false,"Default":true,"Global":true,"Type":"GDPR","UseGoogleVendors":false,"VariantEnabled":false,"TestEndTime":null,"Variants":[],"TemplateName":"Siti Prodotti","Conditions":[],"GCEnable":false,"IsGPPEnabled":false}],"IabData":{"cookieVersion":"1","createdTime":"2023-12-07T08:50:24.417338369","updatedTime":"2023-12-07T08:50:24.417344969","cmpId":"28","cmpVersion":"1","consentScreen":"1","consentLanguage":null,"vendorListVersion":0,"maxVendorId":0,"encodingType":"0","globalVendorListUrl":"https://cdn.cookielaw.org/vendorlist/iabData.json"},"IabV2Data":{"cookieVersion":"1","createdTime":"2023-12-07T08:50:24.471457544","updatedTime":"2023-12-07T08:50:24.471464544","cmpId":"28","cmpVersion":"1","consentScreen":"1","consentLanguage":null,"vendorListVersion":0,"maxVendorId":0,"encodingType":"0","globalVendorListUrl":"https://cdn.cookielaw.org/vendorlist/iab2Data.json"},"Iab2V2Data":{"cookieVersion":"1","createdTime":"2023-12-07T08:50:24.524791880","updatedTime":"2023-12-07T08:50:24.524797840","cmpId":"28","cmpVersion":"1","consentScreen":"1","consentLanguage":null,"vendorListVersion":0,"maxVendorId":0,"encodingType":"0","globalVendorListUrl":"https://cdn.cookielaw.org/vendorlist/iab2V2Data.json"},"GoogleData":{"vendorListVersion":1,"googleVendorListUrl":"https://cdn.cookielaw.org/vendorlist/googleData.json"},"ScriptDynamicLoadEnabled":false,"TenantFeatures":{"CookieV2BannerFocus":true,"CookieV2GPC":true,"CookieV2AssignTemplateRule":true,"CookieV2GeolocationJsonApi":true,"CookieV2TCF21":true,"CookieV2RemoveSettingsIcon":true,"CookieV2BannerLogo":true,"CookieV2GeneralVendors":true,"CookieV2GPP":true},"IsSuppressBanner":false,"IsSuppressPC":false,"PublisherCC":"DE","Domain":"ioapp.it","TenantGuid":"77f17844-04c3-4969-a11d-462ee77acbe1","EnvId":"app-de-prod","RemoteActionsEnabled":false,"GeoRuleGroupName":"Siti Prodotti","GATrackToggle":false,"GATrackAssignedCategory":"","WebFormIntegrationEnabled":false,"WebFormSrcUrl":"","WebFormWorkerUrl":"","GppData":{"cmpId":"28"},"AuthenticatedConsent":false}
```

Comment:
I see `"SkipGeolocation":true` and `"AuthenticatedConsent":false` as big differences. Are they correct?